catalog/app/embed/Dir.js

```diff
@@ -124,14 +124,12 @@ export default function Dir() {
         </div>
         <M.Box flexGrow={1} />
         {!cfg.noDownload && (
-          <FileView.ZipDownloadForm suffix={`dir/${bucket}/${path}`} newTab>
-            <Buttons.Iconized label="Download directory" icon="archive" type="submit" />
-          </FileView.ZipDownloadForm>
+          <Download.Button>
```

Comment:
have you actually tested the embed thing?

Reply:
Yes, though, it was tricky because embed authentication doesn't work well. It fails in most of the cases for me, because it seems there is duplication of the auth calls, and the second call is invalid, because auth code is already used in the first one.

Reply:
hm, that's strange. it was working fine last time i tried

Reply:
I wrote about that in Slack